### PR TITLE
Creates standard Demo styles

### PR DESCRIPTION
**Pull request by jaredgorski**

### CODE REVIEW

File: packages/wd-base/src/_demo.scss [hunk 1]
```diff
@@ -0,0 +1,166 @@
+@import "colors";
```
Comment (pragmaticivan) [inline packages/wd-base/src/_demo.scss]: Can you rename this `_demo.scss` to `demo.scss` it's not a fragment, the outcome of it will generate a .css file.

**Comment by jaredgorski** [comment]
@pragmaticivan Changes made!

File: packages/wd-base/src/demo.scss [hunk 2]
```diff
@@ -0,0 +1,166 @@
+@import "colors";
+@import "typography";
+
+/* ==========================================================================
+   Demo Page
+   ========================================================================== */
+
+/* Reset
+   ========================================================================== */
+* {
```
Comment (pragmaticivan) [inline packages/wd-base/src/demo.scss]: Is that completely effective? or should we use something like https://cssreset.com/scripts/eric-meyer-reset-css/ ?

Comment (jaredgorski) [reply packages/wd-base/src/demo.scss]: @pragmaticivan It works for the purposes of styling the demo pages right now - I added it in order to have a blank slate. I imagine if we want a more detailed reset we can add one easily - what do you think?

Comment (pragmaticivan) [reply packages/wd-base/src/demo.scss]: Sounds good to me.

**Pull request merged**